core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-goals/dot-experiments-configuration-goals.component.html

```diff
@@ -21,9 +21,18 @@ <h2 class="flex align-items-center gap-1 uppercase">
                 <div class="col-4">
                     <p-card dataTestId="card-primary-goal">
                         <ng-template pTemplate="title">
-                            <h3 data-testId="goal-label">
-                                {{ goalTypeMap[vm.goals.primary.type].label | dm }}
-                            </h3>
+                            <div class="flex flex-row justify-content-between">
+                                <h3 data-testId="goal-label">
+                                    {{ goalTypeMap[vm.goals.primary.type].label | dm }}
+                                </h3>
+                                <dot-icon
+                                    class="delete cursor-pointer"
+                                    (click)="deleteGoal($event, 'primary')"
+                                    data-testId="goal-delete-button"
+                                    name="delete"
+                                    size="16"
+                                ></dot-icon>
```

Comment:
We should use an primeng icon button here, not just the icons which is just an `<i>` tag.